app/views/browse/secret.html

```diff
@@ -90,6 +90,9 @@ <h2 class="mar-top-none">
                 </div>
               </div>
             </dl>
+            <p ng-if="!secret.data" class="mar-bottom-xl">
+              There are no data in this secret.
+            </p>
```

Comment:
This is grammatically correct, but reads a little funny to me. Maybe use an empty state message?

```html
<div ng-if="!secret.data" class="empty-state-message text-center">
  <h2>No data.</h2>
  <p>This secret has no data.</p>
</div>
```